src/garage/np/policies/fixed_policy.py

```diff
@@ -0,0 +1,105 @@
+"""Policy that performs a fixed sequence of actions."""
+import numpy as np
+
+from garage.misc.tensor_utils import stack_tensor_dict_list
+from garage.np.policies.base import Policy
+
+
+class FixedPolicy(Policy):
```

Comment:
other frameworks call this `ScriptedPolicy` so I vote we adopt that name.

Reply:
We already have `ScriptedPolicy`, but it does something totally different. We can consider renaming these sometime in the future.